components/cart/CartItem/CartItem.tsx

```diff
@@ -1,15 +1,15 @@
-import { ChangeEvent, FocusEventHandler, useEffect, useState } from 'react'
+import { ChangeEvent, useEffect, useState } from 'react'
 import cn from 'classnames'
 import Image from 'next/image'
 import Link from 'next/link'
 import s from './CartItem.module.css'
-import { Trash, Plus, Minus, Cross } from '@components/icons'
 import { useUI } from '@components/ui/context'
 import type { LineItem } from '@commerce/types/cart'
 import usePrice from '@framework/product/use-price'
 import useUpdateItem from '@framework/cart/use-update-item'
 import useRemoveItem from '@framework/cart/use-remove-item'
 import Quantity from '@components/ui/Quantity'
+import { placeholderImg } from '@components/product/ProductCard'
```

Comment:
I think is better to just duplicate the url as this creates a dependency between ProductCard and CartItem that doesn't really exist, we can later move this url to some constants file and import it from there